libs/dh/shared/feature-authorization/src/lib/dh-signup-mitid.component.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PushModule } from '@rx-angular/template/push';
+import { LetModule } from '@rx-angular/template/let';
+import { OverlayModule } from '@angular/cdk/overlay';
+
+import { WattIconModule } from '@energinet-datahub/watt/icon';
+import { WattSpinnerModule } from '@energinet-datahub/watt/spinner';
```

Comment:
nitpick: doesn't look like `WattIconModule` and `WattSpinnerModule` are used in this component.